lib/engine/corporation.rb

```diff
@@ -26,7 +26,7 @@ class Corporation
                   :type, :floatable, :original_par_price, :reservation_color, :min_price, :ipo_owner,
                   :always_market_price
     attr_reader :companies, :name, :full_name, :fraction_shares, :id, :needs_token_to_par,
-                :presidents_share, :price_multiplier
+                :presidents_share, :price_multiplier, :treasury_as_holding
```

Comment:
what does treasure_as_holding mean

Reply:
Normally, for stock owned by the corporation, corporations distinguish between shares of their own corporation ("treasury" shares) and shares of other corporations ("holdings"). The "treasury_as_holding" is a flag that indicates that all shares are treated the same.

Reply:
Like you said, good naming is hard.